be/src/exec/pipeline/exchange/exchange_sink_operator.cpp

```diff
@@ -309,17 +344,16 @@ Status ExchangeSinkOperator::prepare(RuntimeState* state) {
     }
     RETURN_IF_ERROR(get_block_compression_codec(_compress_type, &_compress_codec));
 
-    std::string instances;
+    std::set<std::string> instances;
     for (const auto& channel : _channels) {
-        if (instances.empty()) {
-            instances = channel->get_fragment_instance_id_str();
-        } else {
-            instances += ", ";
-            instances += channel->get_fragment_instance_id_str();
-        }
+        instances.insert(channel->get_fragment_instance_id_str());
```

Comment:
Good suggestion
But since I have changed the implementations of pipeline level shuffle, and I forget to revert changes here

I'll revert the changes here